pkg/oc/cli/rollback/rollback.go

```diff
@@ -229,8 +227,8 @@ func (o *RollbackOptions) Run() error {
 
 		// A specific deployment was used.
 		target = r
-		configName = appsinternalutil.DeploymentConfigNameFor(obj)
-	case *appsapi.DeploymentConfig:
+		configName = appsutil.DeploymentConfigNameFor(obj)
+	case *appsv1.DeploymentConfig:
```

Comment:
I think you have to change the scheme in the resource builder in `o.findResource` to be `WithScheme(ocscheme.ReadingInternalScheme, ocscheme.ReadingInternalScheme.PrioritizedVersionsAllGroups()...)`, otherwise, you'll still be dealing with internal objects here

Reply:
fixed, thanks!